packages/cactus-common/src/main/typescript/logging/log-helper.ts

```diff
@@ -0,0 +1,100 @@
+import { RuntimeError } from "run-time-error";
+
+export class LogHelper {
+  public static getExceptionStack(exception: unknown): string {
```

Comment:
@m-courtin Could you please put a few code examples in the JSDoc of the methods and the class? Examples that cover how to: 
1) catch + log the error + re-throw with more context (with the usage of `RuntimeError`) 
2) catch + log the error + re-throw as-is
3) catch + conditionally handle it OR re-throw with more context (with the usage of `RuntimeError`)